doc/src/compute_bond_local.rst

```diff
@@ -64,22 +64,32 @@ All these properties are computed for the pair of atoms in a bond,
 whether the two atoms represent a simple diatomic molecule, or are part
 of some larger molecule.
 
-The value *dist* is the current length of the bond.
-The values *dx*, *dy*, and *dz* are the xyz components of the
-*distance* between the pair of atoms. This value is always the
-distance from the atom of lower to the one with the higher id.
+.. versionchanged:: TBD
+
+   The sign of *dx*, *dy*, *dz* is no longer determined by the atom IDs
+   of the bonded atoms but by their order in the bond list to be
+   consistent with *fx*, *fy*, and *fz*.
+
+The value *dist* is the current length of the bond.  The values *dx*,
+*dy*, and *dz* are the :math:`(x,y,z)` components of the distance vector
+:math:`\vec{x_i} - \vec{x_j}` between the atoms in the bond.  The order
+of the atoms is determined by the bond list and the respective atom-IDs
+can be output with :doc:`compute property/local
+<compute_property_local>`.
 
 The value *engpot* is the potential energy for the bond,
 based on the current separation of the pair of atoms in the bond.
 
-The value *force* is the magnitude of the force acting between the
-pair of atoms in the bond.
+The value *force* is the magnitude of the force acting between the pair
+of atoms in the bond, which is positive for a repulsive force and
+negative for an attractive force.
 
-The values *fx*, *fy*, and *fz* are the xyz components of
-*force* between the pair of atoms in the bond. For bond styles that apply
-non-central forces, such as :doc:`bond_style bpm/rotational
-<bond_bpm_rotational>`, these values only include the :math:`(x,y,z)`
-components of the normal force component.
+The values *fx*, *fy*, and *fz* are the :math:`(x,y,z)` components of
```

Comment:
Here and in the pair local doc, I would say it this way:
The values *fx*, *fy*, and *fz* are the :math:`(x,y,z)` components of the force on the first atom I in the bond due to the second atom J in the bond.  Mathematically, they are obtained by multiplying the dx, dy, dz components of the distance vector described above by *force*.  For bond styles that apply non-central ...

Reply:
sorry, it seemed to require my comment twice ...

Reply:
> they are obtained by multiplying the dx, dy, dz components of the distance vector described above by _force_.

That would not be correct. The value of force is the internally computed force multiplied by *r*.
One could say, that it is a vector obtained from multiplying with a unit vector created from dx, dy, dz.

Reply:
yes - good catch - how about "obtained by converting the (dx,dy,dz) distance vector (above) to a unit vector and multiplying by the *force* quantity (above)"